Add Sekoia XDR pack (#34466)
demisto/content: Packs/SekoiaXDR/ReleaseNotes/1_1_0.md

#### Integrations

##### Sekoia XDR

> - Delete Outgoing mirroring function and modify fetch incident function

Reviewer: ```suggestion
- Added mirroring functionality to the integration.
```

#### Layouts

##### Sekoia XDR Layout

> - Use the first layout after rebase

Reviewer: ```suggestion
- Updated the layout to align it with the new mirroring functionality.
```

#### Mappers

##### Sekoia XDR - Incoming Mapper

> - Add some new fileds to mapper

Reviewer: ```suggestion
- Added some new fields to support the mirroring functionality.
```

#### Scripts

##### CloseSekoiaAlert

> - Use the mirroring status to update the alert status
> 
> ##### SekoiaXDRChangeStatus
> 
> - Use the mirroring status to update the alert status

Reviewer: ```suggestion
- Updated the script to align it with the new mirroring functionality.

##### SekoiaXDRChangeStatus

- Updated the script to align it with the new mirroring functionality.
```

> - Updated the script to align it with the new mirroring functionality.

Reviewer: ```suggestion
- Updated the script to align it with the new mirroring functionality.

#### Incident Fields

- New: **SekoiaXDR MirrorOut**
```
